ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -1141,6 +1141,12 @@ literals:
 - potable_hot_leaving_water_temperature_sensor
 - potable_hot_leaving_water_temperature_setpoint
 
+## New fields for tanks
+- level_status:
+ - LOW
```

Comment:
fix this as well, its causing a parsing error. match the indents used for states above.

Reply:
Done